setup-middleware.js

```diff
@@ -13,6 +13,7 @@ function reportViolations(req, res, options) {
   );
 
   ensureDirSync(outputDir);
+  emptyDirSync(outputDir);
```

Comment:
I don't think this is quite right, you can't empty the dir before writing each file (you'd only ever end up with one file in a directory).

I think we need to `ensureDirSync(); emptyDirSync()` down in the `setupMiddleware` function (so it happens once when we get started, not once per POST).

Reply:
Ah yes, you're right. I can fix this up.

Reply:
Fixed. This was a good catch. There's a lot of moving parts here and I need to figure out a way to make it more understandable.